src/lib.rs

```diff
@@ -20,26 +47,35 @@ pub mod prelude {
 use crate::gamepad::GamepadMappingPlugin;
 #[cfg(feature = "gamepad_mapping")]
 pub use crate::gamepad::TouchStickGamepadMapping;
-use crate::input::{
-    send_drag_events_from_mouse, send_drag_events_from_touch, update_sticks_from_drag_events,
-    DragEvent,
-};
+
 pub use crate::{
     behavior::TouchStickType,
     ui::{
         TouchStickInteractionArea, TouchStickUi, TouchStickUiBundle, TouchStickUiKnob,
         TouchStickUiOutline,
     },
 };
+use crate::{
+    input::{
+        send_drag_events_from_mouse, send_drag_events_from_touch, update_sticks_from_drag_events,
+        DragEvent,
+    },
+    ui::TouchStickUiPlugin,
+};
 
 /// pure data, independent of bevy_ui
 #[derive(Component, Clone, Debug, Reflect)]
 #[reflect(Component, Default)]
 pub struct TouchStick<S: StickIdType> {
+    /// type used for identifying this `TouchStick`
```

Comment:
```suggestion
    /// Type used for identifying this [`TouchStick`]
```

This will make TouchStick clickable in the docs. Repeats.